classes/WP_CLI/ActionScheduler_WPCLI_Migration_Command.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+
+namespace Action_Scheduler\WP_CLI;
+
+use Action_Scheduler\Migration\ActionScheduler_MigrationConfig;
+use Action_Scheduler\Migration\ActionScheduler_MigrationRunner;
+use Action_Scheduler\Migration\ActionScheduler_MigrationScheduler;
+use ActionScheduler_Data;
+use WP_CLI;
+use WP_CLI_Command;
+
+/**
+ * Class ActionScheduler_WPCLI_Migration_Command
+ *
+ * @package Action_Scheduler\WP_CLI
+ *
+ * @codeCoverageIgnore
+ */
+class ActionScheduler_WPCLI_Migration_Command extends WP_CLI_Command {
+
+	/**
+	 * Migrates actions to the DB tables store
+	 */
+
+	private $total_processed = 0;
+
+	/**
+	 * Register the command with WP-CLI
+	 */
+	public function register() {
+		if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
```

Comment:
Can `WP_CLI` be disabled in certain contexts where `WP_CLI_Command` may exist? It may be an oversight, but I have a lot of faith in the original author of this, so want to be sure he didn't know something we don't.

Regardless, we may want to move this to here: https://github.com/Prospress/action-scheduler/blob/2.2.5/classes/ActionScheduler.php#L112

That would avoid having two different registration approaches for the two different WP CLI commands.